Dispatcher/Controller/LaunchScreenViewController.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+class LaunchScreenViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        
+        Timer.scheduledTimer(withTimeInterval: 3.0, repeats: false) { timer in
```

Comment:
this timer starts before the user even saw the splash
the timer should be located at viewDidAppear lifecycle function